setup.cfg

```diff
@@ -47,6 +47,3 @@ tests = file: requirements-tests.txt
 
 [options.packages.find]
 where = src
-
-[bdist_wheel]
```

Comment:
this change ensures that the py3-only wheels are built (no `py2` tag in the wheel name)